wcomponents-test-lib/src/main/java/com/github/bordertech/wcomponents/test/selenium/element/SeleniumWTabSetWebElement.java

```diff
@@ -0,0 +1,87 @@
+package com.github.bordertech.wcomponents.test.selenium.element;
+
+import org.apache.commons.lang3.StringUtils;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+
+import java.util.List;
+
+/**
+ * The Wcomponent WTabSet web element.
+ */
+public class SeleniumWTabSetWebElement extends SeleniumWComponentWebElement {
+	/**
+	 * Creates a WebElementWrapper.
+	 *
+	 * @param element the backing element.
+	 * @param driver  the SeleniumWComponentsWebDriver.
+	 */
+	public SeleniumWTabSetWebElement(final WebElement element, final WebDriver driver) {
+		super(element, driver);
+	}
+
+	/**
+	 * Gets the currently selected tabs label.
+	 *
+	 * @return the label value.
+	 */
+	public String findSelectedTabLabel() {
+		WebElement selectedTab = findSelectedTab();
+		if (selectedTab != null) {
+			return selectedTab.getText();
+		}
+		throw new IllegalArgumentException("There is no selected tab.");
+	}
+
+	/**
+	 * Finds the currently selected tab's content.
+	 *
+	 * @return the content of the currently selected tab.
+	 */
+	public SeleniumWComponentWebElement findSelectedTabContent() {
+		WebElement selectedTab = findSelectedTab();
+		if (selectedTab == null) {
+			throw new IllegalArgumentException("There is no selected tab.");
+		}
+		String selectedTabId = selectedTab.getAttribute("id");
```

Comment:
The rest of this method is a little redundant as each tab has an attribute aria-controls which contains the id of its content wrapper. Use that Id to get the tab content.